app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -213,8 +213,9 @@ class HomeFragment : Fragment() {
         if (shouldEnableWallpaper()) {
             wallpapersObserver = WallpapersObserver(
                 appStore = components.appStore,
+                settings = requireContext().settings(),
                 wallpapersUseCases = components.useCases.wallpaperUseCases,
-                wallpaperImageView = binding.wallpaperImageView,
+                wallpaperImageView = binding.wallpaperImageView
```

Comment:
Just a drive-by: Let's keep the trailing comma here. In general, we are encouraging trailing comma and with the ktlint upgrade, we'll also enforce trailing commas.